pkg/operator/controller/ingress/load_balancer_service_test.go

```diff
@@ -674,6 +702,18 @@ func Test_desiredLoadBalancerService(t *testing.T) {
 				localWithFallbackAnnotation:   {true, ""},
 			},
 		},
+		{
+			description:                   "internal load balancer for openstack platform with floating IP",
+			platformStatus:                platformStatus(configv1.OpenStackPlatformType),
+			strategyStatus:                openstackLbWithFloatingIP(operatorv1.InternalLoadBalancer, "1.2.3.4"),
```

Comment:
Shouldn't this be `strategySpec` since we now use spec as desired, and status is actual? It's a valid test, but probably makes more sense to use spec here.